### ISSUE TITLE
Ansible needs to be installed first

### ISSUE THREAD

**Issue by MauricioRoman** [rclayton/vagrant-mysql#2]
It turns out that Ansible needs to be installed for the playbook to run -- I was getting the message "The executable 'ansible-playbook' Vagrant is trying to run was not
found in the PATH variable"

This is documented here:

https://github.com/haikulab/vagrant-statamic-lemp/issues/2

**Comment by rclayton** [comment]
My apologies, I should have made that more obvious.  Yes, I use Ansible as a provisioner.

**Comment by rclayton** [comment]
@MauricioRoman I'm going to close this issue.  I've noted that Ansible is a dependency of this project on the README.md.  Because of the myriad of operating systems and development configurations, I will not be automating the install of Ansible.  If you need help installing Ansible, the documentation on the website is quite thorough: http://docs.ansible.com/intro_installation.html

**Comment by MauricioRoman** [comment]
Thank you Richard,

I have not used Ansible before, so I had to figure out that I had to first
install it in my MacBook before running your scripts, which by the way run
great -- I am already up and running with my MySQL development server.

So my only suggestion would be -- for newbie users like me -- to indicate
that first one must install Ansible before proceeding.

Thank you indeed, for sharing your work!

Mauricio

2015-01-08 11:19 GMT-08:00 Richard Clayton notifications@github.com:
